pkg/controllers/state/suite_test.go

```diff
@@ -1117,6 +1117,12 @@ var _ = Describe("Pod Anti-Affinity", func() {
 })
 
 var _ = Describe("Cluster State Sync", func() {
+	var clusterStateSyncedMetric string
+	var clusterStateNodesCountMetric string
+	BeforeEach(func() {
+		clusterStateSyncedMetric = ExpectFullyQualifiedNameFromCollector(state.ClusterStateSynced)
```

Comment:
Just curious if this is possible: Rather than storing the name and then passing it through to `ExpectMetricGaugeValue`, is there a way to directly pass the gauge into the function so all of this is encapsulated and we never have to store the metric name?

Reply:
We totally could, I honestly don't remember why I opted to go this route at this point. I'm happy to make the change.